.github/workflows/deploy.yml

```diff
@@ -148,6 +142,16 @@ jobs:
           echo "    image: $ECR_REGISTRY/$ECR_REPOSITORY_CMS:$IMAGE_TAG_CMS" >> docker-compose.yml
           echo "    ports:" >> docker-compose.yml
           echo "      - 1337:1337" >> docker-compose.yml
+          echo "  nginx:" >> docker-compose.yml
```

Comment:
I though there was nginx by default at EC2 server and it was setup by conf file at afoco source bundle, but I don't mind to have it as part of docker composer.

Reply:
[there isn't ](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker.container.console.html#docker-software-config) 

> If you manage your Docker environment with Docker Compose, Elastic Beanstalk assumes that you run a proxy server as a container. Therefore it defaults to None for the Proxy server setting, and Elastic Beanstalk does not provide an NGINX configuration.

Reply:
😄 haha, in that case I have no idea how it worked before. I remember checking EC2 created by beanstalk and there was nginx installed. Anyway, if previous solution does not work anymore, this is also good option and basically does same stuff. Maybe Agnieszka will know more...

Reply:
Tiago is convinced I installed it manually and forgot about it. While that sounds totally like me, I'm 99% sure I didn't. But perhaps this has to do with the fact that originally we were using the Dockerrun.aws.json thing, and that's how nginx was preconfigured. Then we switched to docker-compose and I never realised these consequences, because the thing kept working. 🤷‍♀️

Reply:
during my own experiments I was able to get nginx to somehow start and stay up with the default config, so we're not completely crazy here. I suspect nginx may half-baked work, but it's not reliable at all (I rebuilt the current afoco EB stack, and nginx is dead), so the refactor is worth it.